setupRadio.js

```diff
@@ -0,0 +1,30 @@
+function setupRadio(x, y, radioOptions, verticalFlag, defaultOption) {

```

Comment:
The term `options` is generally reserved for associative objects in `js`. Maybe just `labelsArray` and `default`